util/writable_file_io_priority_test.cc

```diff
@@ -0,0 +1,169 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+#include <algorithm>
+
+#include "file/writable_file_writer.h"
+#include "rocksdb/file_system.h"
+#include "test_util/testharness.h"
+#include "test_util/testutil.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+class WritableFileWriterIOPriorityTest : public testing::Test {
```

Comment:
Should we move this whole test suite into file_reader_writer_test.cc, where there exists `class WritableFileWriterTest : public testing::Test {};`  this test can extend from? This should obsolete this new file along with all the added rule to Target/src.mk/etc in this PR?

Reply:
We have many large files with thousands of lines, so I have tried to split them to smaller ones if possible. WritableFileWriterIOPriorityTest doesn't use existing settings of WritableFileWriterTest, so I added a new test file.

Reply:
I'm OK with test-per-file or test-per-feature,  but not really test-per-(file, function argument). util/file_reader_writer_test.cc seems like the right place for this.

Reply:
Since there were two objections to writable_file_io_priority_test.cc, I will change it.

Reply:
Am I looking at the wrong version of the code? This looks unchanged since the comment thread was resolved